src/components/views/Login.js

```diff
@@ -39,8 +40,38 @@ const Login = () => {
   const history = useHistory();
   const [username, setUsername] = useState("");
   const [password, setPassword] = useState("");
+
   let errorResponse = null;
 
+  const pushUser = async () => {
+    try {
+      const id =  localStorage.getItem('id')
+      // if user has no black card yet, server should return null or something.
+      const response = await api.get(`/users/${id}/games/blackCards/current`)
+
+      history.push({
+        pathname: `/game/menu`,
+        state: {
+          id: localStorage.getItem('id'),
+          token: localStorage.getItem('token')
+        }
```

Comment:
Menu does not need state, but it does nothing harmful :)

Reply:
thanks!